packages/studio-be/src/core/dialog/flow/flow-service.ts

```diff
@@ -93,7 +93,7 @@ export class FlowService {
   private _listenForCacheInvalidation() {
     this.cache.events.on('invalidation', async key => {
       try {
-        const matches = key.match(/^([A-Z0-9-_]+)::data\/bots\/([A-Z0-9-_]+)\/flows\/([\s\S]+(flow)\.json)/i)
+        const matches = key.match(/^([A-Z0-9-_]+)::bots\/([A-Z0-9-_]+)\/flows\/([\s\S]+(flow)\.json)/i)
```

Comment:
This is the part that need some clarification. In certain cases, the `data/` part is omitted but I'm not quite sure why.